### DISCUSSION TITLE
Add an element index column with the primary owner of entries in matrix fields

### DISCUSSION THREAD

**Discussion by MoritzLost** [craftcms/cms#16346] [Ideas] [answered]
We have a matrix field that's used for entries that have their own URL. To be able to view all the nested entries directly in one list, we have added a custom source that shows all entries using the entry type that's used in the matrix field. However, there is no way to see at a glance which entry each nested entry belongs to.

It would be nice to have a table column that shows the owner / primary owner of the entry as an element chip, similar to the Ancestors and Parent columns.

**Comment by brandonkelly** [comment, marked as answer]
That was previously requested via #15230. See my response there for why it’s not included yet, and how you can add one yourself.

**Reply by MoritzLost** [reply]
@brandonkelly Thanks for the reply! I missed that thread, I'll go with the solution via hooks then.
